src/components/vanilla/charts/BarChart/components/BarChart.tsx

```diff
@@ -0,0 +1,86 @@
+import { DataResponse, Dimension } from '@embeddable.com/core';
+import {
+  BarElement,
+  CategoryScale,
+  ChartData,
+  Chart as ChartJS,
+  Filler,
+  Legend,
+  LinearScale,
+  PointElement,
+  Title,
+  Tooltip
+} from 'chart.js';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+
+import { COLORS, EMB_FONT, LIGHT_FONT, SMALL_FONT_SIZE, DATE_DISPLAY_FORMATS } from '../../../../constants';
+import formatValue from '../../../../util/format';
+import getBarChartOptions from '../../../../util/getBarChartOptions';
+import Container from '../../../Container';
```

Comment:
It looks like this is not used anywhere in the code

Reply:
Removed